File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/order/exception/InvalidOrderException.java
package band.gosrock.domain.domains.order.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class InvalidOrderException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new InvalidOrderException();

    private InvalidOrderException() {
        super(OrderErrorCode.ORDER_NOT_VALID);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/order/exception/InvalidOrderException.kt
package band.gosrock.domain.domains.order.exception

import band.gosrock.common.exception.DuDoongCodeException

class InvalidOrderException private constructor() : DuDoongCodeException(OrderErrorCode.ORDER_NOT_VALID) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = InvalidOrderException()
    }
}
